07-regression1.Rmd

```diff
@@ -280,7 +294,7 @@ the model and returns the RMSPE for each number of neighbours.
 
```

Comment:
The text above this code chunk wavers back and forth about different aspects of the code, first we say we make a recipe for preprocessing, then we talk about the model spec, and then we go back to talking about the preprocessing... we should keep all the talking about each aspect together so the text and code are easier to read together (fully talk about preprocessing, then talk about the model spec).

Reply:
And then we talk about the model spec again after the code chunk... we should keep all this text organized together.

Reply:
And this text narration says nothing really about the cross validation we are carrying out, we should state explicitly that we are doing this.

Reply:
done all